src/components/Tailwind/Cards/EpochReleaseTimerCard.tsx

```diff
@@ -66,11 +63,6 @@ const EpochReleaseTimerCard = ({ event, content, countdown }: TimeObject) => {
             <p className="text-12px bold ">DEPOSIT IN POOL </p>
             <h4 className="text-lg">{content.deposit}</h4>
           </div>
-          {chainId !== ChainId.MAINNET && (
```

Comment:
Let's preserve this on networks != mainnet, as mentioned on the prior comment

Reply:
@schystz done